valet-key/ValetKey.Web/Controllers/SasController.cs

```diff
@@ -1,32 +1,33 @@
-﻿
-// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-namespace ValetKey.Api.Controllers
-{
-    using System;
-    using System.Configuration;
-    using System.Diagnostics;
-    using System.Net;
-    using System.Net.Http;
-    using System.Web.Http;
-    using Azure.Storage;
-    using Azure.Storage.Blobs;
-    using Azure.Storage.Sas;
-    using Microsoft.Azure;
+﻿using Azure.Storage;
+using Azure.Storage.Blobs;
+using Azure.Storage.Sas;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json;
+using System;
+using System.Diagnostics;
+using System.Net;
+using System.Net.Http;
 
-    public class ValuesController : ApiController
+namespace ValetKey.Web.Controllers
+{
+    [ApiController]
+    public class SasController : ControllerBase
     {
         private readonly BlobServiceClient blobServiceClient;
         private readonly string blobContainer;
+        private IConfiguration configuration;
 
-        public ValuesController()
+
+        public SasController(IConfiguration configuration)
         {
-            this.blobServiceClient = new BlobServiceClient(CloudConfigurationManager.GetSetting("Storage"));
-            this.blobContainer = ConfigurationManager.AppSettings["ContainerName"];
+            this.configuration = configuration;
+            this.blobServiceClient = new BlobServiceClient(configuration.GetSection("AppSettings:StorageConnectionString").Value);
+            this.blobContainer = configuration.GetSection("AppSettings:ContainerName").Value;
         }
 
-        // GET api/Values
-        public StorageEntitySas Get()
+        [HttpGet("Api/Sas")]
```

Comment:
To match how you refer to it everywhere else.  This part of the URL is technically case sensitive (not in practice by default for .net however).  But good to show consistency.

Also, I know the readme calls this out, but I'd feel better if we had something like...

```suggestion
        // This route would typically require authorization
        [HttpGet("api/sas")]
```